modules/ts/include/opencv2/ts.hpp

```diff
@@ -117,7 +117,9 @@
 #endif
 
 #if defined(__OPENCV_BUILD) && defined(__clang__)
+#pragma clang diagnostic push
```

Comment:
Missing push/pop is here for a reason: to avoid massive patching of legacy tests with `override`.

```
$ grep -Rni prepare_to_validation ./modules/ | wc -l
135
```

Reply:
This is strange, but it works for me, I don't get warnings related to `prepare_to_validation` missing override. I tried clang-6, clang-14, clang-17 and only warnings I can see are from ADE.